index.es.md

```diff
@@ -0,0 +1,244 @@
+---
+# Translation instructions are after the "#" character in this first section. (They are comments that do not show up in the web page.)
+title: Introducción a la Accesibilidad Web   # Do not translate "title:". Do translate the text after "title:".
+lang: es   # Change "en" to the translated language shortcode from https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry
+last_updated: 2019-06-26   # Put the date of this translation YYYY-MM-DD (with month in the middle)
+translators:
+- name: "Jorge Rumoroso"   # Replace @@ with translator name
+  link: "https://twitter.com/rumoroso"
+# contributors: #Add one -name: line for every contributor
+# - name: "Contributor Name Here"
+permalink: /fundamentals/accessibility-intro/es   # Add the language shortcode to the end; for example /fundamentals/accessibility-intro/fr
+ref: /fundamentals/accessibility-intro/   # Do not change this
+layout: default
+github:
+  repository: w3c/wai-intro-accessibility
+  branch: gh-pages
+  path: index.es.md   # Add the language shortcode to the middle of the filename, for example index.fr.md
+footer: >   # Translate all the words below, including "Date:" and "Editor:". Do not change these dates.
+  <p><strong>Fecha:</strong> Actualizado a 5 June 2019. Primera publicación en Febrero 2005.</p>
+  <p><strong>Editor:</strong> <a href="http://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Desarrollado por Grupo de Trabajo de Educación y Divulgación (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+# Read Translations Notes at https://github.com/w3c/wai-intro-accessibility/blob/gh-pages/README.md
+# end of translation instructions
+---
+
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Summary" class="full" %}
```

Comment:
Should we translate `title=Summary"` into `title="Resumen"`?

I am translating it manually, but I wonder whether this will be automated or not, because this needs to be done in all the documents.

Reply:
I created [this GitHub issue](https://github.com/w3c/translation-glossaries/issues/3) for you all to agree on translation of "Summary" for these boxes. :-)

You will need to keep doing it manually for now. Maybe eventually we'll automate that, yet not any time soon.

Thanks.